service/logger/audit/utils.go

```diff
@@ -35,39 +50,81 @@ type auditEventObject struct {
 	Attributes eventObjectAttributes `json:"attributes,omitempty"`
 }
 
+func (e auditEventObject) LogValue() slog.Value {
+	return slog.GroupValue(
+		slog.String("type", e.Type.String()),
+		slog.String("id", e.ID),
+		slog.String("name", e.Name),
+		slog.Any("attributes", e.Attributes))
+}
+
 // event.object.attributes
 type eventObjectAttributes struct {
 	Assertions  []string `json:"assertions,omitempty"`
 	Attrs       []string `json:"attrs,omitempty"`
 	Permissions []string `json:"permissions,omitempty"`
 }
 
+func (e eventObjectAttributes) LogValue() slog.Value {
+	return slog.GroupValue(
+		slog.String("assertions", strings.Join(e.Assertions, ",")),
+		slog.String("attrs", strings.Join(e.Attrs, ",")),
+		slog.String("permissions", strings.Join(e.Permissions, ",")))
```

Comment:
This won't escape the strings (e.g. commas will cause problems if we want to parse it back). Since the String valuer will escape the quotes, this shouldn't be too much of an issue, but maybe you will be better served with

```suggestion
		slog.Any("assertions", e.Assertions),
		slog.Any("attrs", e.Attrs),
		slog.Any("permissions", e.Permissions))
```